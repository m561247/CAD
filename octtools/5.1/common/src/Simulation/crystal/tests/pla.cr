input *inp *En
output *out
delay *in 0 0
