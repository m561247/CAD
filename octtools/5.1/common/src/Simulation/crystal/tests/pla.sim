| units: 200	tech: nmos
e 118 GND 92 2 4 119 -42
e 64 GND 118 2 4 32 -44
d 118 Vdd 118 4.5 2 11 -44
e 111 92 GND 2 4 119 -50
e 32 111 GND 2 4 72 -52
d 111 Vdd 111 4.5 2 11 -52
e 97 GND 89 2 4 95 -58
e 32 97 GND 2 4 72 -60
e 30 97 GND 2 4 56 -60
e 28 97 GND 2 4 40 -60
e 26 97 GND 2 4 24 -60
d 97 Vdd 97 4.5 2 11 -60
e EnableOut 92 54 2 2 120 -66
e EnableOut 91 58 2 2 112 -66
e EnableOut 90 53 2 2 104 -66
e EnableOut 89 57 2 2 96 -66
e 32 GND 66 2 8 72 -79
e 30 GND 65 2 8 56 -79
e 28 GND 64 2 8 40 -79
e 26 GND 63 2 8 24 -79
d 66 66 Vdd 2 2 66 -82
d 65 65 Vdd 2 2 50 -82
d 64 64 Vdd 2 2 34 -82
d 63 63 Vdd 2 2 18 -82
e 58 GND output3 2 8 113 -86
e 57 GND output1 2 8 97 -86
d 32 Vdd 32 2 2 66 -90
d 30 Vdd 30 2 2 50 -90
d 28 Vdd 28 2 2 34 -90
d 26 Vdd 26 2 2 18 -90
e 54 output4 GND 2 8 118 -92
e 53 output2 GND 2 8 102 -92
d output3 output3 Vdd 4 2 115 -102
d output1 output1 Vdd 4 2 99 -102
d output4 output4 Vdd 4 2 116 -110
d output2 output2 Vdd 4 2 100 -110
e 9 GND 32 2 16 72 -110
e 8 GND 30 2 16 56 -110
e 7 GND 28 2 16 40 -110
e 6 GND 26 2 16 24 -110
e EnableIn 9 input4 2 2 72 -120
e EnableIn 8 input3 2 2 56 -120
e EnableIn 7 input2 2 2 40 -120
e EnableIn 6 input1 2 2 24 -120
d 92 Vdd 92 4.5 2 119 -12
d 91 Vdd 91 4.5 2 111 -12
d 90 Vdd 90 4.5 2 103 -12
d 89 Vdd 89 4.5 2 95 -12
e 143 91 GND 2 4 111 -18
e 66 GND 143 2 4 64 -20
e 65 GND 143 2 4 48 -20
d 143 Vdd 143 4.5 2 11 -20
e 134 GND 90 2 4 103 -26
e 64 GND 134 2 4 32 -28
d 134 Vdd 134 4.5 2 11 -28
e 127 92 GND 2 4 119 -34
e 63 GND 127 2 4 16 -36
d 127 Vdd 127 4.5 2 11 -36
C output1 GND 51
C output2 GND 52
C output3 GND 51
C output4 GND 52
C 26 GND 57
C 28 GND 57
C 30 GND 57
C 32 GND 60
C Vdd GND 432
C 64 GND 51
C 65 GND 54
C 66 GND 54
C 90 GND 57
C 92 GND 71
C 97 GND 100
C 111 GND 73
C 118 GND 73
C 127 GND 63
C 134 GND 67
C 143 GND 82
