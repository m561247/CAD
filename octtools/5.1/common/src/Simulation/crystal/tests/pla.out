1*******08/20/83 ********  SPICE 2G.6    3/15/83 ********14:51:15*****

0SPICE DECK GENERATED BY CRYSTAL.                                                

0****     INPUT LISTING                    TEMPERATURE =   27.000 DEG C

0***********************************************************************




 ** THE FOLLOWING LINES DEFINE THE NMOS MODELS AND VDD.  

 .MODEL E NMOS(VTO=.9 NSUB=2E15 TOX=.1U UO=800 XQC=.5    
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .MODEL D NMOS(VTO=-3.2 NSUB=2E15 TOX=.1U UO=800 XQC=.5  
  +LEVEL=2 VMAX=1E5 CGSO=245P CGDO=245P) 
 .OPTIONS NOMOD  
 .WIDTH OUT=80   
 VDD 1 0 5       
 VBB 2 0 0       
 .WIDTH OUT=100  
 * NODES 4-5 CORRESPOND TO OUTPUT4 (SEE GATE AT 116,-110)
 * NODES 6-7 CORRESPOND TO 54 (SEE GATE AT 118,-92)      
 * NODES 8-9 CORRESPOND TO 92 (SEE GATE AT 119,-12)      
 * NODES 10-11 CORRESPOND TO 118 (SEE GATE AT 11,-44)    
 * NODES 12-13 CORRESPOND TO 64 (SEE GATE AT 32,-28)     
 * NODES 14-15 CORRESPOND TO 28 (SEE GATE AT 34,-90)     
 * NODES 16-17 CORRESPOND TO 7 (SEE GATE AT 40,-110)     
 * NODES 18-19 CORRESPOND TO INPUT2 (SEE DRAIN AT 40,-120)       
         
 C1 11 0 0.073PF 
 C2 9 0 0.075PF  
 C3 5 0 0.052PF  
 C4 15 0 0.074PF 
 C5 13 0 0.068PF 
 M1 0 7 5 2 E L=4.0U W=16.0U     
 M2 1 5 5 2 D L=8.0U W=4.0U      
 M3 7 1 9 2 E L=4.0U W=4.0U      
 M4 9 9 1 2 D L=9.0U W=4.0U      
 M5 9 11 0 2 E L=4.0U W=8.0U     
 M6 11 13 0 2 E L=4.0U W=8.0U    
 M7 11 11 1 2 D L=9.0U W=4.0U    
 M8 1 13 13 2 D L=4.0U W=4.0U    
 M9 13 15 0 2 E L=4.0U W=16.0U   
 M10 15 17 0 2 E L=4.0U W=32.0U  
 M11 15 15 1 2 D L=4.0U W=4.0U   
 M12 19 1 17 2 E L=4.0U W=4.0U   
         
 * INITIAL CONDITIONS:   
 .IC V(17)=0     
 .IC V(11)=5     
 .IC V(7)=0      
 .IC V(9)=0      
 .IC V(5)=5      
 .IC V(15)=5     
 .IC V(13)=0     
 .IC V(19)=0     
         
 VIN 19 0 PULSE(0 5 0NS 0NS 0NS) 
 .TRAN .1NS 20NS 
 .PRINT TRAN V(17) V(15) V(13) V(11) V(7) V(5) (0,5)     
 .END    
1****************08/20/83 ************************  SPICE 2G.6    3/15/83 ************************14:51:15****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 INITIAL TRANSIENT SOLUTION                           TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************


  NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


 (  1)    5.0000    (  2)    0.        (  5)    5.0000    (  7)    0.0000    (  9)    0.0001

 ( 11)    5.0000    ( 13)    0.0001    ( 15)    5.0000    ( 17)    0.0000    ( 19)    0.    





     VOLTAGE SOURCE CURRENTS

     NAME       CURRENT


     VDD      -1.894d-04

     VBB       7.627d-11

     VIN       5.454d-31


     TOTAL POWER DISSIPATION   9.47d-04  WATTS
1****************08/20/83 ************************  SPICE 2G.6    3/15/83 ************************14:51:15****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 OPERATING POINT INFORMATION                          TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************


0
0**** MOSFETS


0            M1        M2        M3        M4        M5        M6        M7        M8        M9      
0MODEL     E         D         E         D         E         E         D         D         E       
 ID        0.  d+00  1.93d-12 -5.94d-09 -5.24d-05  1.19d-08  1.93d-12  1.93d-12  1.37d-04  6.20d-08
 VGS         -5.000     0.        5.000    -5.000     5.000     0.000    -0.000     0.        5.000
 VDS         -5.000     0.000    -0.000    -5.000     0.000     5.000    -0.000     5.000     0.000
 VBS         -5.000    -5.000    -0.000    -5.000     0.        0.       -5.000    -0.000     0.   


0            M10       M11       M12     
0MODEL     E         D         E       
 ID        1.93d-12  1.93d-12 -5.55d-31
 VGS          0.000    -0.000     5.000
 VDS          5.000    -0.000    -0.000
 VBS          0.       -5.000    -0.000
1****************08/20/83 ************************  SPICE 2G.6    3/15/83 ************************14:51:15****************

0SPICE DECK GENERATED BY CRYSTAL.                                                
0****                 TRANSIENT ANALYSIS                                   TEMPERATURE =   27.000 DEG C

0*************************************************************************************************************************



     TIME       V(17)       V(15)       V(13)       V(11)       V(7)        V(5)        
X
 
  0.   d+00     5.452d-31   5.000d+00   1.369d-04   5.000d+00   5.936d-09   5.000d+00
  1.000d-10     2.554d-01   5.028d+00   1.137d-01   5.006d+00   1.352d-03   5.001d+00
  2.000d-10     5.227d-01   5.052d+00   1.814d-01   5.010d+00   7.580d-03   5.002d+00
  3.000d-10     7.894d-01   5.072d+00   2.238d-01   5.012d+00   2.465d-02   5.003d+00
  4.000d-10     1.022d+00   5.085d+00   2.500d-01   5.013d+00   5.046d-02   5.004d+00
  5.000d-10     1.207d+00   5.088d+00   2.658d-01   5.015d+00   7.395d-02   5.006d+00
  6.000d-10     1.359d+00   5.077d+00   2.739d-01   5.015d+00   9.438d-02   5.007d+00
  7.000d-10     1.491d+00   5.055d+00   2.783d-01   5.016d+00   1.128d-01   5.007d+00
  8.000d-10     1.590d+00   5.008d+00   2.791d-01   5.016d+00   1.272d-01   5.008d+00
  9.000d-10     1.690d+00   4.961d+00   2.800d-01   5.016d+00   1.415d-01   5.009d+00
  1.000d-09     1.775d+00   4.902d+00   2.810d-01   5.016d+00   1.539d-01   5.009d+00
  1.100d-09     1.837d+00   4.821d+00   2.821d-01   5.016d+00   1.628d-01   5.009d+00
  1.200d-09     1.898d+00   4.740d+00   2.832d-01   5.016d+00   1.717d-01   5.009d+00
  1.300d-09     1.960d+00   4.659d+00   2.844d-01   5.016d+00   1.805d-01   5.009d+00
  1.400d-09     2.013d+00   4.569d+00   2.870d-01   5.016d+00   1.879d-01   5.009d+00
  1.500d-09     2.050d+00   4.463d+00   2.923d-01   5.016d+00   1.928d-01   5.009d+00
  1.600d-09     2.087d+00   4.357d+00   2.975d-01   5.016d+00   1.976d-01   5.009d+00
  1.700d-09     2.125d+00   4.251d+00   3.027d-01   5.016d+00   2.025d-01   5.009d+00
  1.800d-09     2.157d+00   4.141d+00   3.095d-01   5.016d+00   2.065d-01   5.009d+00
  1.900d-09     2.182d+00   4.025d+00   3.190d-01   5.016d+00   2.092d-01   5.009d+00
  2.000d-09     2.206d+00   3.909d+00   3.285d-01   5.016d+00   2.118d-01   5.009d+00
  2.100d-09     2.230d+00   3.793d+00   3.380d-01   5.015d+00   2.145d-01   5.008d+00
  2.200d-09     2.252d+00   3.675d+00   3.493d-01   5.015d+00   2.167d-01   5.008d+00
  2.300d-09     2.270d+00   3.555d+00   3.638d-01   5.015d+00   2.181d-01   5.008d+00
  2.400d-09     2.287d+00   3.436d+00   3.782d-01   5.015d+00   2.196d-01   5.008d+00
  2.500d-09     2.304d+00   3.316d+00   3.927d-01   5.015d+00   2.210d-01   5.007d+00
  2.600d-09     2.320d+00   3.196d+00   4.098d-01   5.015d+00   2.228d-01   5.007d+00
  2.700d-09     2.333d+00   3.077d+00   4.315d-01   5.015d+00   2.249d-01   5.007d+00
  2.800d-09     2.346d+00   2.958d+00   4.533d-01   5.015d+00   2.271d-01   5.006d+00
  2.900d-09     2.358d+00   2.838d+00   4.751d-01   5.016d+00   2.293d-01   5.006d+00
  3.000d-09     2.370d+00   2.720d+00   5.018d-01   5.016d+00   2.305d-01   5.006d+00
  3.100d-09     2.381d+00   2.605d+00   5.372d-01   5.016d+00   2.301d-01   5.006d+00
  3.200d-09     2.391d+00   2.489d+00   5.726d-01   5.016d+00   2.296d-01   5.005d+00
  3.300d-09     2.401d+00   2.373d+00   6.079d-01   5.017d+00   2.292d-01   5.005d+00
  3.400d-09     2.411d+00   2.259d+00   6.535d-01   5.017d+00   2.289d-01   5.005d+00
  3.500d-09     2.419d+00   2.147d+00   7.169d-01   5.018d+00   2.289d-01   5.005d+00
  3.600d-09     2.427d+00   2.035d+00   7.802d-01   5.019d+00   2.290d-01   5.004d+00
  3.700d-09     2.436d+00   1.923d+00   8.436d-01   5.019d+00   2.290d-01   5.004d+00
  3.800d-09     2.444d+00   1.814d+00   9.176d-01   5.020d+00   2.288d-01   5.004d+00
  3.900d-09     2.451d+00   1.708d+00   1.010d+00   5.019d+00   2.283d-01   5.004d+00
  4.000d-09     2.458d+00   1.603d+00   1.102d+00   5.019d+00   2.277d-01   5.004d+00
  4.100d-09     2.466d+00   1.497d+00   1.195d+00   5.018d+00   2.272d-01   5.004d+00
  4.200d-09     2.473d+00   1.396d+00   1.291d+00   5.015d+00   2.268d-01   5.003d+00
  4.300d-09     2.481d+00   1.303d+00   1.394d+00   5.006d+00   2.266d-01   5.003d+00
  4.400d-09     2.488d+00   1.210d+00   1.497d+00   4.997d+00   2.264d-01   5.003d+00
  4.500d-09     2.496d+00   1.117d+00   1.600d+00   4.988d+00   2.262d-01   5.003d+00
  4.600d-09     2.501d+00   1.032d+00   1.702d+00   4.973d+00   2.258d-01   5.003d+00
  4.700d-09     2.503d+00   9.615d-01   1.802d+00   4.947d+00   2.252d-01   5.003d+00
  4.800d-09     2.505d+00   8.907d-01   1.903d+00   4.921d+00   2.246d-01   5.003d+00
  4.900d-09     2.507d+00   8.198d-01   2.003d+00   4.895d+00   2.240d-01   5.003d+00
  5.000d-09     2.509d+00   7.591d-01   2.101d+00   4.861d+00   2.236d-01   5.002d+00
  5.100d-09     2.511d+00   7.159d-01   2.193d+00   4.812d+00   2.235d-01   5.002d+00
  5.200d-09     2.513d+00   6.727d-01   2.286d+00   4.763d+00   2.234d-01   5.002d+00
  5.300d-09     2.515d+00   6.294d-01   2.379d+00   4.714d+00   2.233d-01   5.002d+00
  5.400d-09     2.519d+00   5.938d-01   2.468d+00   4.656d+00   2.232d-01   5.002d+00
  5.500d-09     2.526d+00   5.713d-01   2.554d+00   4.583d+00   2.232d-01   5.002d+00
  5.600d-09     2.533d+00   5.488d-01   2.639d+00   4.509d+00   2.231d-01   5.002d+00
  5.700d-09     2.540d+00   5.262d-01   2.724d+00   4.435d+00   2.231d-01   5.002d+00
  5.800d-09     2.548d+00   5.076d-01   2.806d+00   4.352d+00   2.233d-01   5.002d+00
  5.900d-09     2.557d+00   4.957d-01   2.884d+00   4.253d+00   2.240d-01   5.002d+00
  6.000d-09     2.566d+00   4.837d-01   2.962d+00   4.155d+00   2.247d-01   5.002d+00
  6.100d-09     2.575d+00   4.718d-01   3.040d+00   4.056d+00   2.254d-01   5.002d+00
  6.200d-09     2.585d+00   4.616d-01   3.116d+00   3.948d+00   2.272d-01   5.002d+00
  6.300d-09     2.594d+00   4.544d-01   3.187d+00   3.825d+00   2.310d-01   5.002d+00
  6.400d-09     2.603d+00   4.472d-01   3.258d+00   3.702d+00   2.347d-01   5.002d+00
  6.500d-09     2.613d+00   4.400d-01   3.329d+00   3.579d+00   2.384d-01   5.002d+00
  6.600d-09     2.622d+00   4.336d-01   3.398d+00   3.448d+00   2.427d-01   5.002d+00
  6.700d-09     2.631d+00   4.286d-01   3.463d+00   3.304d+00   2.479d-01   5.003d+00
  6.800d-09     2.640d+00   4.236d-01   3.527d+00   3.159d+00   2.531d-01   5.003d+00
  6.900d-09     2.649d+00   4.186d-01   3.591d+00   3.014d+00   2.583d-01   5.003d+00
  7.000d-09     2.657d+00   4.141d-01   3.653d+00   2.864d+00   2.647d-01   5.003d+00
  7.100d-09     2.666d+00   4.103d-01   3.710d+00   2.703d+00   2.734d-01   5.004d+00
  7.200d-09     2.674d+00   4.065d-01   3.767d+00   2.543d+00   2.820d-01   5.004d+00
  7.300d-09     2.682d+00   4.026d-01   3.824d+00   2.383d+00   2.907d-01   5.004d+00
  7.400d-09     2.690d+00   3.991d-01   3.878d+00   2.222d+00   3.018d-01   5.005d+00
  7.500d-09     2.697d+00   3.959d-01   3.926d+00   2.060d+00   3.173d-01   5.006d+00
  7.600d-09     2.704d+00   3.927d-01   3.974d+00   1.898d+00   3.327d-01   5.006d+00
  7.700d-09     2.712d+00   3.895d-01   4.022d+00   1.736d+00   3.482d-01   5.007d+00
  7.800d-09     2.719d+00   3.865d-01   4.066d+00   1.583d+00   3.681d-01   5.008d+00
  7.900d-09     2.725d+00   3.839d-01   4.104d+00   1.444d+00   3.955d-01   5.009d+00
  8.000d-09     2.732d+00   3.813d-01   4.142d+00   1.306d+00   4.230d-01   5.010d+00
  8.100d-09     2.739d+00   3.787d-01   4.180d+00   1.168d+00   4.505d-01   5.012d+00
  8.200d-09     2.745d+00   3.763d-01   4.216d+00   1.046d+00   4.820d-01   5.013d+00
  8.300d-09     2.751d+00   3.741d-01   4.249d+00   9.504d-01   5.208d-01   5.015d+00
  8.400d-09     2.757d+00   3.719d-01   4.281d+00   8.553d-01   5.595d-01   5.017d+00
  8.500d-09     2.763d+00   3.698d-01   4.313d+00   7.602d-01   5.982d-01   5.018d+00
  8.600d-09     2.769d+00   3.677d-01   4.344d+00   6.798d-01   6.377d-01   5.020d+00
  8.700d-09     2.775d+00   3.659d-01   4.373d+00   6.250d-01   6.783d-01   5.021d+00
  8.800d-09     2.780d+00   3.641d-01   4.402d+00   5.702d-01   7.190d-01   5.023d+00
  8.900d-09     2.786d+00   3.623d-01   4.431d+00   5.154d-01   7.597d-01   5.024d+00
  9.000d-09     2.791d+00   3.606d-01   4.459d+00   4.703d-01   7.991d-01   5.026d+00
  9.100d-09     2.796d+00   3.590d-01   4.485d+00   4.421d-01   8.366d-01   5.027d+00
  9.200d-09     2.801d+00   3.574d-01   4.511d+00   4.140d-01   8.740d-01   5.028d+00
  9.300d-09     2.806d+00   3.558d-01   4.537d+00   3.858d-01   9.114d-01   5.029d+00
  9.400d-09     2.811d+00   3.543d-01   4.561d+00   3.627d-01   9.486d-01   5.029d+00
  9.500d-09     2.815d+00   3.529d-01   4.583d+00   3.486d-01   9.852d-01   5.029d+00
  9.600d-09     2.820d+00   3.515d-01   4.605d+00   3.344d-01   1.022d+00   5.028d+00
  9.700d-09     2.825d+00   3.501d-01   4.627d+00   3.202d-01   1.058d+00   5.028d+00
  9.800d-09     2.829d+00   3.488d-01   4.647d+00   3.086d-01   1.095d+00   5.025d+00
  9.900d-09     2.833d+00   3.476d-01   4.666d+00   3.013d-01   1.130d+00   5.021d+00
  1.000d-08     2.837d+00   3.464d-01   4.684d+00   2.940d-01   1.166d+00   5.016d+00
  1.010d-08     2.842d+00   3.452d-01   4.702d+00   2.867d-01   1.202d+00   5.011d+00
  1.020d-08     2.846d+00   3.440d-01   4.719d+00   2.806d-01   1.237d+00   5.003d+00
  1.030d-08     2.850d+00   3.429d-01   4.734d+00   2.766d-01   1.271d+00   4.991d+00
  1.040d-08     2.853d+00   3.418d-01   4.749d+00   2.726d-01   1.305d+00   4.978d+00
  1.050d-08     2.857d+00   3.407d-01   4.763d+00   2.686d-01   1.339d+00   4.965d+00
  1.060d-08     2.861d+00   3.397d-01   4.777d+00   2.651d-01   1.372d+00   4.950d+00
  1.070d-08     2.865d+00   3.387d-01   4.789d+00   2.628d-01   1.405d+00   4.928d+00
  1.080d-08     2.868d+00   3.378d-01   4.801d+00   2.604d-01   1.437d+00   4.907d+00
  1.090d-08     2.872d+00   3.368d-01   4.813d+00   2.580d-01   1.470d+00   4.886d+00
  1.100d-08     2.875d+00   3.359d-01   4.824d+00   2.559d-01   1.501d+00   4.861d+00
  1.110d-08     2.879d+00   3.350d-01   4.834d+00   2.544d-01   1.532d+00   4.830d+00
  1.120d-08     2.882d+00   3.341d-01   4.843d+00   2.528d-01   1.563d+00   4.799d+00
  1.130d-08     2.885d+00   3.333d-01   4.853d+00   2.513d-01   1.594d+00   4.768d+00
  1.140d-08     2.889d+00   3.324d-01   4.862d+00   2.499d-01   1.624d+00   4.733d+00
  1.150d-08     2.892d+00   3.317d-01   4.869d+00   2.488d-01   1.653d+00   4.692d+00
  1.160d-08     2.895d+00   3.309d-01   4.877d+00   2.478d-01   1.683d+00   4.651d+00
  1.170d-08     2.898d+00   3.301d-01   4.884d+00   2.467d-01   1.712d+00   4.610d+00
  1.180d-08     2.901d+00   3.294d-01   4.891d+00   2.457d-01   1.740d+00   4.565d+00
  1.190d-08     2.904d+00   3.287d-01   4.897d+00   2.449d-01   1.768d+00   4.514d+00
  1.200d-08     2.907d+00   3.280d-01   4.903d+00   2.442d-01   1.796d+00   4.462d+00
  1.210d-08     2.909d+00   3.273d-01   4.909d+00   2.434d-01   1.823d+00   4.411d+00
  1.220d-08     2.912d+00   3.266d-01   4.915d+00   2.427d-01   1.850d+00   4.356d+00
  1.230d-08     2.915d+00   3.259d-01   4.919d+00   2.421d-01   1.876d+00   4.295d+00
  1.240d-08     2.919d+00   3.252d-01   4.924d+00   2.415d-01   1.902d+00   4.234d+00
  1.250d-08     2.922d+00   3.245d-01   4.929d+00   2.409d-01   1.928d+00   4.173d+00
  1.260d-08     2.925d+00   3.239d-01   4.933d+00   2.404d-01   1.954d+00   4.108d+00
  1.270d-08     2.927d+00   3.233d-01   4.937d+00   2.399d-01   1.978d+00   4.037d+00
  1.280d-08     2.929d+00   3.227d-01   4.940d+00   2.395d-01   2.003d+00   3.966d+00
  1.290d-08     2.932d+00   3.221d-01   4.944d+00   2.390d-01   2.027d+00   3.895d+00
  1.300d-08     2.935d+00   3.215d-01   4.948d+00   2.386d-01   2.051d+00   3.820d+00
  1.310d-08     2.938d+00   3.209d-01   4.950d+00   2.383d-01   2.073d+00   3.739d+00
  1.320d-08     2.941d+00   3.203d-01   4.953d+00   2.379d-01   2.096d+00   3.658d+00
  1.330d-08     2.944d+00   3.197d-01   4.956d+00   2.376d-01   2.119d+00   3.577d+00
  1.340d-08     2.947d+00   3.191d-01   4.959d+00   2.372d-01   2.141d+00   3.493d+00
  1.350d-08     2.949d+00   3.185d-01   4.961d+00   2.370d-01   2.162d+00   3.402d+00
  1.360d-08     2.952d+00   3.179d-01   4.964d+00   2.367d-01   2.183d+00   3.311d+00
  1.370d-08     2.954d+00   3.173d-01   4.966d+00   2.364d-01   2.203d+00   3.220d+00
  1.380d-08     2.957d+00   3.167d-01   4.968d+00   2.362d-01   2.224d+00   3.126d+00
  1.390d-08     2.960d+00   3.161d-01   4.970d+00   2.360d-01   2.243d+00   3.026d+00
  1.400d-08     2.963d+00   3.156d-01   4.972d+00   2.357d-01   2.262d+00   2.926d+00
  1.410d-08     2.966d+00   3.151d-01   4.973d+00   2.355d-01   2.281d+00   2.827d+00
  1.420d-08     2.969d+00   3.145d-01   4.975d+00   2.353d-01   2.300d+00   2.725d+00
  1.430d-08     2.970d+00   3.141d-01   4.977d+00   2.351d-01   2.317d+00   2.619d+00
  1.440d-08     2.972d+00   3.136d-01   4.978d+00   2.348d-01   2.335d+00   2.513d+00
  1.450d-08     2.974d+00   3.131d-01   4.980d+00   2.346d-01   2.353d+00   2.407d+00
  1.460d-08     2.976d+00   3.126d-01   4.981d+00   2.344d-01   2.370d+00   2.301d+00
  1.470d-08     2.979d+00   3.122d-01   4.982d+00   2.343d-01   2.386d+00   2.192d+00
  1.480d-08     2.982d+00   3.117d-01   4.983d+00   2.342d-01   2.402d+00   2.083d+00
  1.490d-08     2.985d+00   3.113d-01   4.984d+00   2.341d-01   2.418d+00   1.974d+00
  1.500d-08     2.987d+00   3.108d-01   4.985d+00   2.340d-01   2.434d+00   1.866d+00
  1.510d-08     2.989d+00   3.104d-01   4.986d+00   2.339d-01   2.448d+00   1.757d+00
  1.520d-08     2.990d+00   3.100d-01   4.987d+00   2.338d-01   2.463d+00   1.649d+00
  1.530d-08     2.992d+00   3.096d-01   4.988d+00   2.337d-01   2.478d+00   1.540d+00
  1.540d-08     2.993d+00   3.092d-01   4.989d+00   2.337d-01   2.492d+00   1.434d+00
  1.550d-08     2.995d+00   3.089d-01   4.989d+00   2.336d-01   2.505d+00   1.330d+00
  1.560d-08     2.996d+00   3.086d-01   4.990d+00   2.335d-01   2.518d+00   1.226d+00
  1.570d-08     2.998d+00   3.083d-01   4.991d+00   2.334d-01   2.531d+00   1.123d+00
  1.580d-08     2.999d+00   3.080d-01   4.991d+00   2.333d-01   2.541d+00   1.028d+00
  1.590d-08     3.001d+00   3.078d-01   4.992d+00   2.332d-01   2.546d+00   9.463d-01
  1.600d-08     3.002d+00   3.075d-01   4.993d+00   2.331d-01   2.551d+00   8.651d-01
  1.610d-08     3.004d+00   3.072d-01   4.993d+00   2.329d-01   2.556d+00   7.838d-01
  1.620d-08     3.005d+00   3.070d-01   4.994d+00   2.328d-01   2.560d+00   7.143d-01
  1.630d-08     3.006d+00   3.067d-01   4.994d+00   2.327d-01   2.565d+00   6.651d-01
  1.640d-08     3.008d+00   3.064d-01   4.995d+00   2.326d-01   2.570d+00   6.159d-01
  1.650d-08     3.009d+00   3.061d-01   4.996d+00   2.325d-01   2.575d+00   5.667d-01
  1.660d-08     3.011d+00   3.059d-01   4.996d+00   2.324d-01   2.582d+00   5.262d-01
  1.670d-08     3.012d+00   3.056d-01   4.996d+00   2.324d-01   2.592d+00   5.006d-01
  1.680d-08     3.014d+00   3.053d-01   4.997d+00   2.324d-01   2.602d+00   4.751d-01
  1.690d-08     3.015d+00   3.050d-01   4.997d+00   2.325d-01   2.612d+00   4.495d-01
  1.700d-08     3.017d+00   3.047d-01   4.997d+00   2.325d-01   2.623d+00   4.285d-01
  1.710d-08     3.018d+00   3.044d-01   4.997d+00   2.325d-01   2.634d+00   4.155d-01
  1.720d-08     3.020d+00   3.041d-01   4.997d+00   2.325d-01   2.646d+00   4.024d-01
  1.730d-08     3.021d+00   3.039d-01   4.998d+00   2.325d-01   2.658d+00   3.894d-01
  1.740d-08     3.023d+00   3.036d-01   4.998d+00   2.325d-01   2.670d+00   3.784d-01
  1.750d-08     3.024d+00   3.033d-01   4.998d+00   2.325d-01   2.682d+00   3.712d-01
  1.760d-08     3.025d+00   3.030d-01   4.998d+00   2.325d-01   2.693d+00   3.639d-01
  1.770d-08     3.027d+00   3.027d-01   4.998d+00   2.325d-01   2.705d+00   3.567d-01
  1.780d-08     3.028d+00   3.025d-01   4.999d+00   2.325d-01   2.716d+00   3.505d-01
  1.790d-08     3.030d+00   3.023d-01   4.999d+00   2.324d-01   2.727d+00   3.459d-01
  1.800d-08     3.031d+00   3.021d-01   4.999d+00   2.324d-01   2.737d+00   3.413d-01
  1.810d-08     3.033d+00   3.019d-01   4.999d+00   2.324d-01   2.748d+00   3.368d-01
  1.820d-08     3.034d+00   3.017d-01   4.999d+00   2.324d-01   2.758d+00   3.327d-01
  1.830d-08     3.035d+00   3.015d-01   4.999d+00   2.324d-01   2.767d+00   3.295d-01
  1.840d-08     3.036d+00   3.013d-01   4.999d+00   2.323d-01   2.776d+00   3.263d-01
  1.850d-08     3.037d+00   3.011d-01   4.999d+00   2.323d-01   2.786d+00   3.231d-01
  1.860d-08     3.038d+00   3.009d-01   4.999d+00   2.323d-01   2.794d+00   3.202d-01
  1.870d-08     3.040d+00   3.007d-01   4.999d+00   2.322d-01   2.803d+00   3.178d-01
  1.880d-08     3.041d+00   3.005d-01   4.999d+00   2.322d-01   2.811d+00   3.153d-01
  1.890d-08     3.043d+00   3.002d-01   4.999d+00   2.322d-01   2.819d+00   3.129d-01
  1.900d-08     3.044d+00   3.000d-01   4.999d+00   2.321d-01   2.827d+00   3.107d-01
  1.910d-08     3.045d+00   2.998d-01   4.999d+00   2.321d-01   2.834d+00   3.087d-01
  1.920d-08     3.046d+00   2.996d-01   4.999d+00   2.320d-01   2.841d+00   3.068d-01
  1.930d-08     3.047d+00   2.994d-01   4.999d+00   2.320d-01   2.848d+00   3.049d-01
  1.940d-08     3.048d+00   2.992d-01   4.999d+00   2.320d-01   2.855d+00   3.031d-01
  1.950d-08     3.049d+00   2.990d-01   4.999d+00   2.319d-01   2.861d+00   3.015d-01
  1.960d-08     3.050d+00   2.988d-01   4.999d+00   2.319d-01   2.867d+00   3.000d-01
  1.970d-08     3.052d+00   2.986d-01   4.999d+00   2.319d-01   2.873d+00   2.984d-01
  1.980d-08     3.053d+00   2.984d-01   5.000d+00   2.319d-01   2.879d+00   2.969d-01
  1.990d-08     3.054d+00   2.982d-01   5.000d+00   2.318d-01   2.885d+00   2.956d-01
  2.000d-08     3.055d+00   2.980d-01   5.000d+00   2.318d-01   2.891d+00   2.942d-01
Y
0
         JOB CONCLUDED
0         TOTAL JOB TIME           54.52
